kuams3_teleop:
  ros__parameters:
    axis_linear: 1       # The joystick axis index used for controlling linear velocity (e.g., forward/backward)
    axis_angular: 0      # The joystick axis index used for controlling angular velocity (e.g., left/right)
    axis_deadman: 5      # The button index used for the deadman switch
    scale_linear: 0.3    # Scale factor for linear velocity (maximum speed)
    scale_angular: 0.9   # Scale factor for angular velocity (maximum angular speed)
